### ISSUE TITLE
Should use accessibilityIdentifier instead of accesibilityLabel

### ISSUE THREAD

**Issue by piercifani** [kif-framework/KIF#577]
In order to be consistent with the use of accessibility elements, KIF should use ```accessibilityIdentifier``` tu identify views in the hierarchy instead of ```accessibilityLabel``` in order to being able to localise the accessibility information of each element

**Comment by ohenak** [comment]
I actually have the same question too and wondering why we decided to go with accessibilityLabel. Is it due to the Apple internal API that KIF is leveraging?

I am more willing to help updating it to support accessibilityIdentifier.

**Comment by bnickel** [comment]
You can read my rants on the subject in issue https://github.com/kif-framework/KIF/issues/243.

**tl;dr** I don't believe accessibilityIdentifier is really functional testing, localization is a test environment problem and can be pinned in your scheme, the tests are less readable, and I've actually already implemented this in the `KIF/IdentifierTests` subspec but I don't like it so I've never advertised it (and I guess no one else has either).

I suppose someone should do a PR to get instructions for the identifier tests on README.
